src/browser/modules/D3Visualization/components/Inspector.jsx

```diff
@@ -50,7 +50,7 @@ const mapItemProperties = itemProperties =>
           {prop.key + ': '}
         </StyledInspectorFooterRowListKey>
         <StyledInspectorFooterRowListValue className='value'>
-          {prop.value.toString()}
+          {prop.value ? prop.value.toString() : prop.value}
```

Comment:
What happens if `prop.value` is `0` or `false`?
I'd suggest a stricter check: 
```js
![null, undefined].includes(prop.value) && typeof prop.value.toString === 'function'
``` 
or similar.